_data/master/navbars/getting-started.yml

```diff
@@ -80,7 +84,7 @@ toc:
   - title: Installation
     section:
     - title: Requirements
-      path: /getting-started/mesos/installation/prerequisites
+      path: /getting-started/mesos/installation/reqs/system   
```

Comment:
trailing spaces here

Reply:
Hmmmmm...can't see them ❓

Reply:
There are three spaces after `/reqs/system` - highlighted in dark green.